Projects/private-ca/roles/vault/tasks/main.yml

```diff
@@ -51,8 +55,6 @@
     command: vault login -client-key /etc/ssl/cert-key.pem -client-cert /etc/ssl/cert.pem
     responses:
       hidden: "{{ vault_output.stdout_lines[6][20:] }}"
-    echo: yes
-  run_once: true
```

Comment:
We actually need to log in to all nodes because we need to unseal the vault instance on each host.